app/helpers/samples_helper.rb

```diff
@@ -142,14 +143,26 @@ def parsed_samples_for_s3_path(s3_path, project_id, host_genome_id)
     default_attributes = { project_id: project_id.to_i,
                            host_genome_id: host_genome_id,
                            status: 'created', }
-    s3_path.chomp!('/')
-    s3_bucket = 's3://' + s3_path.sub('s3://', '').split('/')[0]
-    s3_output, _stderr, status = Open3.capture3("aws", "s3", "ls", "--recursive", "#{s3_path}/")
-    return unless status.exitstatus.zero?
-    s3_output.chomp!
-    entries = s3_output.split("\n").reject { |line| line.include? "Undetermined" }
```

Comment:
it looks the removal of "Undetermined" files was dropped. was this intentional? I think it's still useful to ignore "Undetermined" files if we've been using this

Reply:
Are you sure this is about a file name pattern? I mean, it gives me the impression this is something related to parsing the result lines of `aws s3 ls` that could contain such thing, but I might be totally wrong. Could you give me an example in which cases this is needed?

Reply:
Thank you for the explanation. I believed it was related to AWS cli outputs. I'm adding it back.